components/Tabs/Tabs.js

```diff
@@ -1,38 +1,37 @@
-
 class TabItem {
   constructor(element) {
-    // attach dom element to object. Example in Tabs class
+    this.element = element;
   }
 
   select() {
-    // should use classList
+    this.element.classList.add('Tabs__item-selected');// should use classList
```

Comment:
Add a space between the ending semicolon and the comment

Reply:
Thanks so much for the feedback. I’m pretty active on the slack, asking questions, etc. I am starting to get a little better at committing more often. Also, thanks for letting me know about the space.